src/lib/killableMonsters.ts

```diff
@@ -130,7 +130,7 @@ const killableMonsters: KillableMonster[] = [
 		difficultyRating: 8,
 		itemsRequired: ['Armadyl chestplate', 'Armadyl chainskirt'],
 		notifyDrops: ['Vorki', 'Jar of decay', 'Draconic visage', 'Skeletal visage'],
-		qpRequired: 0
+		qpRequired: 200
```

Comment:
these 2 `qpRequired` changes might be more appropriate for a future PR, I want to give people a few days warning (Starting today) to start grinding QP now if they want to keep killing them, so theres hopefully less crying when its done